File: beginner/__3000__/3342/3342.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scannner in = new Scanner(System.in);

        int n = in.nexInt();

        int total_cases = n * n;
        int b = 0; 
        int p = 0;

        if (n % 2 == 0){
            b = total_cases / 2;
            p = total_cases / 2;
        } else {
            b = (total_cases + 1) / 2;
            p = b - 1;
        }

        System.out.println(b + " casas brancas e " p + " casas pretas");
        in.close();
    }
}

File: beginner/__3000__/3342/3342.ts
import * as fs from "fs"

const input = fs.readFIleSync("/dev/stdin", "utf8")

function keanuChess(input: number): void {
    
    let total_cases = input * input;
    let b = 0, p = 0

    if (input % 2 == 0){
        b = total_cases / 2;
        p = total_cases / 2;
    } else {
        b = (total_cases + 1) / 2;
        p = b - 1
    }
    console.log(`${b} casas brancas e ${p} casas pretas`)
}

keanuChess(Number(input))